packages/kit/src/runtime/client/router.js

```diff
@@ -225,7 +221,7 @@ export class Router {
 				keepfocus,
 				chain,
 				url.hash,
-				state,
+				{},
```

Comment:
if you're removing it, I'd think it would be removed from `_navigate` as well?

Reply:
ah yep, i thought it was used in the other cases for scroll management, but evidently not. i think it's needed in #3293 but there's already going to be a merge conflict there anyway